apps/modernization-api/src/main/java/gov/cdc/nbs/controller/PatientController.java

```diff
@@ -68,8 +68,8 @@ public Optional<Person> findPatientById(@Argument Long id) {
 
     @MutationMapping()
     @PreAuthorize(DELETE_PATIENT)
```

Comment:
It looks like legacy requires also requires the `View Patient` permission when doing deletes

Image from System Admin -> manage permissions screen
![image](https://user-images.githubusercontent.com/109251240/220440553-d4aa3d7c-15eb-40ed-ba18-240baba67dab.png)